### PR TITLE
14 feature 14 create usersurveydatainterface entity layer and survey class related interface

### PR DESCRIPTION
**Pull request by GagandeepLubana**

### CODE REVIEW

File: app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/classes/Survey.java
Comment (realdevvora) [inline app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/classes/Survey.java:1]: just make sure to add these methods onto the CRC document, and its good

**Review by realdevvora — APPROVED** [review]
👍

**Review by JaspreetKhela — CHANGES_REQUESTED** [review]
Please add comments to your code. Everything else looks good.

**Review by JaspreetKhela — APPROVED** [review]

**Review by uyoyo0 — APPROVED** [review]
Looks good

**Review by realdevvora — APPROVED** [review]

**Pull request merged**